__tests__/utils.test.js

```diff
@@ -54,103 +52,199 @@ describe("Northcoders News API", () => {
+    it("200 - responds with an articles array of articles objects with no body and in decending order", () => {
+      return request(app)
+        .get("/api/articles")
+        .expect(200)
+        .then(({ body }) => {
+          body.article.map((key) => {
+            expect(key.hasOwnProperty("body")).toBe(false);
+            expect(key.hasOwnProperty("comment_count")).toBe(true);
+          });
+          expect(body.article.length).toBe(5);
+          expect(body.article).toBeSortedBy("created_at", { descending: true });
+        });
+    });
+  });
 
+  describe("GET /api/articles/:article_id/comments", () => {
+    it("200 - respond with an array of properties for the given ID", () => {
+      return request(app)
+        .get("/api/articles/9/comments")
+        .expect(200)
+        .then(({ body }) => {
+          expect(body.article.length).toBe(2);
+          expect(body.article).toBeSortedBy("created_at");
+          body.article.map((keys) => {
+            expect(keys.hasOwnProperty("randomKey")).toBe(false);
+            expect(keys.hasOwnProperty("comment_id")).toBe(true);
+            expect(keys.hasOwnProperty("votes")).toBe(true);
+            expect(keys.hasOwnProperty("created_at")).toBe(true);
+            expect(keys.hasOwnProperty("author")).toBe(true);
+            expect(keys.hasOwnProperty("body")).toBe(true);
+            expect(keys.hasOwnProperty("article_id")).toBe(true);
+          });
+        });
+    });
 
+    it("404 - respond stating that the id does not exists", () => {
+      return request(app)
+        .get("/api/articles/1000/comments")
+        .expect(404)
+        .then(({ body }) => {
+          expect(body.msg).toBe("No article found");
+        });
+    });
 
+    it("400 - respond with a bad request message", () => {
+      return request(app)
+        .get("/api/articles/one/comments")
+        .expect(400)
+        .then(({ body }) => {
+          expect(body.msg).toBe("Bad request");
+        });
+    });
+  });
 
+  describe("CORE: POST /api/articles/:article_id/comments", () => {
+    it("201 - respond with the new comment that is posted and the ID matches as well", () => {
+      const newComt = {
+        username: "icellusedkars",
+        body: "No comment...",
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComt)
+        .expect(201)
+        .then(({ body }) => {
+          expect(body.comment.body).toBe(newComt.body);
+          expect(body.comment.article_id).toBe(1);
+        });
+    });
 
+    it("201 - check if the username and body exists or not", () => {
+      const newComt = {
+        username: "icellusedkars",
+        body: "No comment...",
+      };
+      return request(app)
+        .post("/api/articles/1/comments")
+        .send(newComt)
+        .expect(201)
+        .then(({ body }) => {
+          expect("author" in body.comment).toBe(true);
+          expect("body" in body.comment).toBe(true);
```

Comment:
We are duplicating some assertions from the test before here, so no need to include this test. If a test auto passes we should always consider why 👍